### PR TITLE
Upgrade to Ratpack 1.4.1

### PR DESCRIPTION
**Pull request by jeff-blaisdell**

### CODE REVIEW

File: ratpack-bearer-auth/src/test/groovy/st/ratpack/auth/BearerTokenAuthHandlerSpec.groovy [hunk 1]
```diff
@@ -22,7 +22,6 @@ class BearerTokenAuthHandlerSpec extends Specification {
 
 		where:
 		authHeader << [
-			null,
```
Comment (jeff-blaisdell) [inline ratpack-bearer-auth/src/test/groovy/st/ratpack/auth/BearerTokenAuthHandlerSpec.groovy:25]: Ratpack fixture now prevents nulls, so test not needed.

File: ratpack-bearer-auth/src/main/java/st/ratpack/auth/springsec/SpringSecCheckTokenValidator.java [hunk 2]
```diff
@@ -93,7 +92,7 @@ private String buildBasicAuthHeader(String user, String password) {
 
 	private static ObjectMapper buildObjectMapper() {
 		ObjectMapper objectMapper = new ObjectMapper();
-		objectMapper.setPropertyNamingStrategy(PropertyNamingStrategy.CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES);
+		objectMapper.setPropertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE);
```
Comment (jeff-blaisdell) [inline ratpack-bearer-auth/src/main/java/st/ratpack/auth/springsec/SpringSecCheckTokenValidator.java:95]: Fix deprecation warning.

File: ratpack-bearer-auth/src/main/java/st/ratpack/auth/springsec/SpringSecCheckTokenValidator.java [hunk 3]
```diff
@@ -54,7 +53,7 @@ public SpringSecCheckTokenValidator(SpringSecCheckAuthModule.Config config, Http
 			});
 		});
 
-		return Promise.of(downstream ->
+		return Promise.async(downstream ->
```
Comment (jeff-blaisdell) [inline ratpack-bearer-auth/src/main/java/st/ratpack/auth/springsec/SpringSecCheckTokenValidator.java:56]: Fix deprecation warning.

File: version.txt [hunk 4]
```diff
@@ -1 +1 @@
-2.0.1-SNAPSHOT
+1.4.1.1-SNAPSHOT
```
Comment (beckje01) [inline version.txt]: We want to keep the versioning separate from ratpack if possible still.

**Comment by jeff-blaisdell** [comment]
Reverted back to previous versioning scheme

**Review by beckje01 — APPROVED** [review]

**Pull request closed without merging**